src/views/Verify/index.view.tsx

```diff
@@ -22,7 +24,9 @@ const VerifyView: React.FC = () => {
   return (
     <>
       <div className='verify-view'>
-        <p>You are not a verified user</p>
+        <NavBar theme='home' />
```

Comment:
If we do the team page theme, can we pass 'team' to navbar?

Reply:
Tested that and it looks better, this change has been made!